wording for workloads. Added Autopatch notes (#3140)
MicrosoftDocs/memdocs: memdocs/configmgr/comanage/workloads.md

# Co-management workloads

> You don't have to switch any of the workloads, but you can do them individually or several or all at once when you're ready. However, until you switch the workloads over to Intune, Microsoft Endpoint Configuration Manager continues to manage those workloads that you don't switch to Intune, alongside all other features of Configuration Manager that co-management doesn't support.

Reviewer: ```suggestion
You don't have to switch any of the workloads, but you can do them individually or several or all at the same time when you're ready. However, until you switch the workloads over to Intune, Microsoft Endpoint Configuration Manager continues to manage those workloads that you don't switch to Intune, along with all other features of Configuration Manager that co-management doesn't support.
```

> If you switch a workload to Intune, but later change your mind, you can switch it back to Configuration Manager, although there maye be an impact for example Windows and Office versions will remain at a later version if installed by Intune.

Reviewer: ```suggestion
If you switch a workload to Intune, but later change your mind, you can switch it back to Configuration Manager, although there might be an impact, for example, Windows and Office versions will remain at a later version if installed by Intune.
```

## Windows Update policies

> Windows Update for Business policies let you configure deferral policies for Windows 10 or later feature updates or quality updates for Windows 10 or later devices managed directly by Windows Update for Business. Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.

Reviewer: ```suggestion
Windows Update for Business policies let you configure deferral policies for Windows 10 or later feature updates or quality updates for Windows 10 or later devices managed directly by Windows Update for Business. Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```

The device configuration workload includes settings that you manage for devices in your organization. Switching this workload also moves the **Resource Access** and **Endpoint Protection** workloads.

> You can still deploy settings from Configuration Manager to co-managed devices even though Intune is the device configuration authority. This exception might be used to configure settings that your organization requires but aren't yet available in Intune. Specify this exception on a [Configuration Manager configuration baseline](../compliance/deploy-use/create-configuration-baselines.md). Enable the option to **Always apply this baseline even for co-managed clients** when creating the baseline. You can change it later on the **General** tab of the properties of an existing baseline. Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.

Reviewer: ```suggestion
You can still deploy settings from Configuration Manager to co-managed devices even though Intune is the device configuration authority. This exception might be used to configure settings that your organization requires but aren't yet available in Intune. Specify this exception on a [Configuration Manager configuration baseline](../compliance/deploy-use/create-configuration-baselines.md). Enable the option to **Always apply this baseline even for co-managed clients** when creating the baseline. You can change it later on the **General** tab of the properties of an existing baseline. Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```

- [Use Update Channel and Target Version settings to update Microsoft 365 with Microsoft Intune Administrative Templates](../../intune/configuration/administrative-templates-update-office.md)
- [Manage Microsoft 365 Apps with Configuration Manager](../sum/deploy-use/manage-office-365-proplus-updates.md).

> Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.

Reviewer: ```suggestion
Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```
